src/theory/quantifiers/trigger.h

```diff
@@ -33,59 +33,89 @@ namespace inst {
 class IMGenerator;
 class InstMatchGenerator;
 
-/** Information about a node used in a trigger. */
+/** Information about a node used in a trigger. 
```

Comment:
Great!  I now understand this code.  To improve the presentation a bit, give a summary here at the top of the class, like:

Information stored includes:
1) the free variables; and
2) information about which terms are useful for matching